### ISSUE TITLE
VS2015 unresolved extern build error using supplied telnet libs

### ISSUE THREAD

**Issue by Fish-Git** [Fish-Git/hyperion#2]
Attempting to build Hyperion using VS2015 (with the default `telnet32.lib` or `telnet64.lib` libraries that now come delivered with the repository as of commit bd78204c55f33dce9957a095edcb6e80911c12c3) results in an "unresolved external symbol" link error for `vscprintf` and `vsnprintf_s`:

```
  [...]
  console.c
     Creating library msvc.AMD64.obj\hdt3270.lib and object msvc.AMD64.obj\hdt3270.exp
telnet64.lib(telnet.c.obj) : error LNK2001: unresolved external symbol __imp__vscprintf
telnet64.lib(telnet.c.obj) : error LNK2001: unresolved external symbol __imp__vsnprintf_s
msvc.AMD64.bin\hdt3270.dll : fatal error LNK1120: 2 unresolved externals
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\x86_amd64\link.EXE"' : return code '0x460'
  Stop.
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.MakeFile.Targets(46,5): error MSB3073: The command "makefile.bat RETAIL-X64 makefile.msvc 64 -a -title "** The unofficial SoftDevLabs version of Hercules **"" exited with code 2.
```

**Comment by Fish-Git** [comment]
Am looking into it...

**Comment by Fish-Git** [comment]
Fixed by 561bfee1ee492b51b6b221dc5d1ab5eb115b41b3
